### PR TITLE
Infotip for small screens

### PR DESCRIPTION
**Pull request by raveenakothapally**
ISSUE #459

PLEASE SQUASH.

Similar to https://github.com/eBay/skin/pull/1094

But moved styles under <strike>`dialog--mini`</strike> `dialog__window--mini` modifier.

### CODE REVIEW

**Comment by ianmcburnie** [comment]
Thanks. I will compare and contrast the two different PRs and approaches.

**Comment by ianmcburnie** [comment]
I'm going to make one quick change and shuffle these new selectors and rules under a `dialog__window--mini` selector instead.

**Comment by ianmcburnie** [comment]
@agliga @raveenakothapally 

I shuffled the rules under `dialog__window.dialog__window--mini` modifier. I left in the chained selector, but ideally would like to investigate and remove that at some point (close button position gets broken when removed).

Added a quick js fix for modal behaviour on non-transitioning dialogs too.

Screen Shot 2020-05-11 at 10 24 33 AM

Screen Shot 2020-05-11 at 10 29 33 AM

File: docs/static/ds6/docs.min.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ $_mod.def("/makeup-screenreader-trap$0.1.4/index",function(e,t,n,r,a){"use stric
...
...))}},{key:"destroy",value:function(){this._destroyed=!0,this.unobserveEvents(),this._onCloseButtonClickListener=null,this._onConfirmButtonClickListener=null,this._onRejectButtonClickListener=null,this._onKeyDownListener=null,this._onOpenTransitionEndListener=null,this._onCloseTransitionEndListener=null}},{key:"focusables",get:function(){return m(this._windowEl)}},{key:"open",get:function(){return!this._el.hidden},set:function(t){!0===t&&!0!==this.open?this._hasTransitions?(this.unobserveEvents(),this._cancelTransition&&this._cancelTransition(),this._cancelTransition=p(this._el,"".concat(this._options.dialogBaseClass,"--show"),this._onOpenTransitionEndListener)):(this._el.hidden=!1,this._isModal&&(r(this),E.modal(this._el)),this._el.dispatchEvent(new CustomEvent("".concat(this._options.dialogBaseClass,"-open")))):!1===t&&!1!==this.open&&(this._hasTransitions?(this.unobserveEvents(),this._cancelTransition&&this._cancelTransition(),this._cancelTransition=p(this._el,"".concat(this._options.dialogBaseClass,"--hide"),this._onCloseTransitionEndListener)):(E.unmodal(),this._el.hidden=!0,this._el.dispatchEvent(new CustomEvent("".concat(this._options.dialogBaseClass,"-close")))))}}]),t}()});
```
Comment (agliga) [inline docs/static/ds6/docs.min.js:123]: Why did this change. I dont think it should

Comment (ianmcburnie) [reply docs/static/ds6/docs.min.js:123]: I updated the `dialog.js` class.

**Pull request merged**